addons/TBMod_nachschub/configs/CfgVehicles_USA.hpp

```diff
@@ -51,15 +51,9 @@ class TB_supply_usa_unterlauf : WRAPPER_NAME(Box_IND_Support_F)
     class TransportMagazines
     {
         // Explosiv
-        MACRO_ADDMAGAZINE(rhs_mag_M397_HET,5);
-        MACRO_ADDMAGAZINE(rhs_mag_M441_HE,5);
-        MACRO_ADDMAGAZINE(rhs_mag_M433_HEDP,3);
-
-        // Smokes
-        MACRO_ADDMAGAZINE(rhs_mag_m714_White,4);
-        MACRO_ADDMAGAZINE(rhs_mag_m713_Red,2);
-        MACRO_ADDMAGAZINE(rhs_mag_m715_Green,2);
-        MACRO_ADDMAGAZINE(rhs_mag_m716_yellow,2);
+        MACRO_ADDMAGAZINE(TB_rhs_mag_40mm_HE,5);
```

Comment:
Denke mal das können wir leicht hochschrauben... 5 HE sind wenig... in einer Kiste sollen ja nicht nur einer was kriegen können.

Reply:
Gerne! Ich hatte nur nicht die Zeit zu testen, wie viel man maximal reinmachen kann, bevor die Kiste zu schwer ist. Gerne noch mehr Granaten rein :)